.vsts-dotnet-ci.yml

```diff
@@ -248,3 +248,9 @@ jobs:
       ArtifactName: 'MonoOnMac test logs'
     continueOnError: true
     condition: always()
+
+- template: /eng/common/templates/job/source-build.yml
+  parameters:
+    platform:
+      name: 'Managed'
+      container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-3e800f1-20190501005343'
```

Comment:
This seems very specific; is there a mechanism to keep it updated?

Reply:
I logged an issue to refactor the source-build templates to provide this as a default - https://github.com/dotnet/source-build/issues/2182